flow/Hostname.actor.cpp

```diff
@@ -83,10 +83,25 @@ ACTOR Future<Void> resolveImpl(Hostname* self) {
 	return Void();
 }
 
+ACTOR Future<Void> resolveWithRetryImpl(Hostname* self) {
+	loop {
+		try {
+			wait(resolveImpl(self));
+			return Void();
+		} catch (...) {
+			wait(delay(FLOW_KNOBS->HOSTNAME_RESOLVE_DELAY));
```

Comment:
You need to throw cancellation error if that's the error. Like this: https://github.com/apple/foundationdb/blob/aaaf42525ae48381e49d152c43f4eb8782d5c831/fdbrpc/FlowTransport.actor.cpp#L790

Otherwise, what will happen is that the loop will continue run, but the ACTOR is destroyed, which will cause segfault.